resources/scss/views/ItemList/Components/Filter/item-filter-list.scss

```diff
@@ -1,8 +1,99 @@
-.findologic-filter-wrapper #filterCollapse {
-    position: relative;
-    z-index: 0;
-}
+@import '../../../../Findologic';
```

Comment:
This line will have to be removed once everything is imported in `findologic.scss`.